src/api/spec/models/token/service_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Token::Service do
+  let(:user) { create(:user, login: 'foo') }
+  let(:token) { create(:service_token, user: user) }
+  let(:time_now) { Time.now.utc.to_s }
```

Comment:
You can drop the `Timecop.freeze` and etc, if you just define the time as something deterministic like:

```
let(:time_now) { Time.new('2021-11-02 12:41:33 +0100').utc.to_s }
```

Reply:
@vpereira but how do I control which time is set in the token object if I dont use the freeze?

Reply:
I mean I get that I can use a static date and time, but somehow I need to ensure that the same time is set in the `triggered_at` field after the token got triggered. Afraid of flickering tests if I dont use the freeze

Reply:
Without `Timecop`, you'll still have to mock the value returned by `Time.zone.now`. Using `Timecop` is fine.

Reply:
You shouldn't care about the specific value but if the behavior is there. I.e is the value changing, is the value other than nil, etc.

Reply:
If you really want to test the specific time, you should go with https://api.rubyonrails.org/v5.2.6/classes/ActiveSupport/Testing/TimeHelpers.html

Reply:
@vpereira please have another look, I now only check if the `triggered_at` field changed, not checking for a particular date